package.json

```diff
@@ -25,12 +25,14 @@
   },
   "dependencies": {
     "classnames": "^2.2.6",
+    "foundation-sites": "^6.5.0-rc.2",
     "gh-pages": "1.2.0",
     "prop-types": "15.6.2",
     "react": "16.4.1",
     "react-collapse": "^4.0.3",
     "react-dom": "16.4.1",
     "react-motion": "^0.5.2",
+    "react-router-dom": "^4.3.1",
```

Comment:
In some projects, we are not using react-router anymore so I think we could get rid of this.

Reply:
👍